manifests/c/CEAEDFOCCT/SALOME/9.8.0/CEAEDFOCCT.SALOME.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: CEAEDFOCCT.SALOME
+PackageVersion: 9.8.0
+PackageLocale: en-US
+Publisher: CEA EDF OCCT
+PackageName: SALOME
+License: LGPL-2.1-or-later
+ShortDescription: SALOME Setup

```

Comment:
Can you please provide a better description of what SALOME is?

Reply:
I have updated the short description